mobile-widgets/qml/DiveDetailsEdit.qml

```diff
@@ -76,14 +81,18 @@ Item {
 		usedCyl[2] != null ? usedCyl[2] = cylinderBox2.currentText : null
 		usedCyl[3] != null ? usedCyl[3] = cylinderBox3.currentText : null
 		usedCyl[4] != null ? usedCyl[4] = cylinderBox4.currentText : null
-
+		usedGas[0] != null ? usedGas[0] = txtGasMix0.text : null
+		usedGas[1] != null ? usedGas[1] = txtGasMix1.text : null
+		usedGas[2] != null ? usedGas[2] = txtGasMix2.text : null
+		usedGas[3] != null ? usedGas[3] = txtGasMix3.text : null
+		usedGas[4] != null ? usedGas[4] = txtGasMix4.text : null
```

Comment:
ternary operator usage not clear/usual/correct.

Reply:
Ugh that's convenient but... let's not write Perl code here, please

Reply:
Yeah, that one needs to be addressed.